suite-common/formatters/src/utils/convertCryptoToFiatAmount.ts

```diff
@@ -0,0 +1,25 @@
+import { formatNetworkAmount, toFiatCurrency } from '@suite-common/wallet-utils';
```

Comment:
unit tests should always follow same folder sturcutre as files they are testing structure, so test file for this file should be in folder: `/utils/tests/...` same goes for formatter, their tests should be in `formatters/tests/...`

Reply:
fixed